### ISSUE TITLE
File share from ownCloud import not working

### ISSUE THREAD

**Issue by plinss** [strukturag/nextcloud-spreedme#13]
This appears to be a CSP issue. 

When in a call click the "Share file" button, then click the "ownCloud Import" button. 

Result: the "Please select the file(s) you want to share" dialog appears, with a brief loading icon, then goes blank. On Firefox the console logs: 'Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://<server> 'unsafe-eval'").'

Spreed.ME tool v0.1.4 installed on ownCloud v9.0.0, using spreed-webrtc v0.25.3 server running in same origin.
Other functions including screen sharing appear to be working normally.

**Comment by leonklingele** [comment]
Unfortunately I was not able to reproduce this.
Are running spreed-webrtc or ownCloud over http (instead of https)?

**Comment by plinss** [comment]
I'm running over https, interestingly I can no longer reproduce it either. I was having some issues with my turn server setup that are resolved (BTW, the turn server instructions could be expanded upon a bit, I'll send a PR if I get the chance), since that was sorted the file share seems to be working (I don't think I changed anything else since). Oddly, when I saw this issue it was during A/V test calls on the same network that were working without the turn server.

Feel free to close this, if I get a chance I'll try to come up with a better recipe to reproduce and re-open if I find it.
